ecommerce_api_client/tests/test_client.py

```diff
@@ -20,7 +21,7 @@ class EcommerceApiClientTests(TestCase):
 
     def test_valid_configuration(self):
         """ The constructor should return successfully if all arguments are valid. """
```

Comment:
Related to my previous comment, the constructor should also return successfully if `FULL_NAME` and `EMAIL` are equal to the empty string, since these are valid arguments. I suggest adding a test case to cover this scenario.